### ISSUE TITLE
Type stability and /=

### ISSUE THREAD

**Issue by matthieugomez** [tonyhffong/Lint.jl#97]
Lint returns type warnings for constructions like this

```julia
a = 1.0
a /= 3
```
This returns
```
WARN   Previously used a has apparent type Float64, but now assigned Int64
```

**Comment by tonyhffong** [comment]
I think it's better now. Thanks for the report.
